modules/install-booting-from-an-iso-over-http-redfish.adoc

```diff
@@ -21,6 +21,7 @@ Ensure that you have the latest firmware version of iDRAC that is compatible wit
 .Prerequisites
 
 * Download the installation {op-system-first} ISO.
+* Use iDRAC 9 based Dell PowerEdge server.
```

Comment:
"based" is placed here among a string of adjectives, so insetad of hyphenating the term to `-iDRAC-9-based Dell PowerEdge server`, how about `Use an iDRAC 9 X that is compatible with a Dell PowerEdge server`. WDYT?

Reply:
Changed to
> Use a Dell PowerEdge server that is compatible with iDRAC9.

Reply:
Yeah, your update is much better. LGTM!